cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerClusterHealthService.java

```diff
@@ -54,10 +65,17 @@ public class ClouderaManagerClusterHealthService implements ClusterHealthService
 
     static final String HOST_AGENT_CERTIFICATE_EXPIRY = "HOST_AGENT_CERTIFICATE_EXPIRY";
 
+    // TODO Do we extend this to include additional health check params?
+    static final String NODE_MANAGER_CONNECTIVITY = "NODE_MANAGER_CONNECTIVITY";
+
+    static final String YARN = "yarn";
+
+    static final String NODEMANAGER = "NODEMANAGER";
+
     private static final Logger LOGGER = LoggerFactory.getLogger(ClouderaManagerClusterHealthService.class);
 
     private static final Set<ApiHealthSummary> IGNORED_HEALTH_SUMMARIES = Sets.immutableEnumSet(
-            ApiHealthSummary.DISABLED,
+            DISABLED,
             ApiHealthSummary.NOT_AVAILABLE,
             ApiHealthSummary.HISTORY_NOT_AVAILABLE
```

Comment:
You can static import NOT_AVAILABLE and HISTORY_NOT_AVAILABLE

Reply:
Done